homeassistant/components/tibber/sensor.py

```diff
@@ -496,7 +496,7 @@ def __init__(
     @property
     def available(self) -> bool:
         """Return True if entity is available."""
-        return self._tibber_home.rt_subscription_running
+        return bool(self._tibber_home.rt_subscription_running)
```

Comment:
This was needed to pass the MyPy tests

Reply:
The mypy tests have already been passing... So, this change seems incorrect and probably related to your development environment?